### PR TITLE
Master

### PR DESCRIPTION
**Pull request by kevinsmia1939**
Use GNUInstallDirs for directories. Fix for this #1852.
I tried using openMVG pipeline to check if it work, it does work, but check to make sure.

Kind regards

### CODE REVIEW

**Comment by pmoulon** [comment]
Thank you. I will take a look during the week and let you know. FYI when you run install there is a manifest file that trace all the file that have been install. By looking to it you can check if there is any file going in the wrong path.

**Comment by kevinsmia1939** [comment]
> Thank you. I will take a look during the week and let you know. FYI when you run install there is a manifest file that trace all the file that have been install. By looking to it you can check if there is any file going in the wrong path.

Hello,

I check all the files, it went to the correct path.
I only apply gnuinstalldir to lib only, I left bin and include alone because I don't know if I will break them, but if you want me to cover them too let me know.

**Comment by pmoulon** [comment]
Let's do step by step. Starting with the LIB is a good start

File: src/openMVG/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+include(GNUInstallDirs)
```
Comment (pmoulon) [inline src/openMVG/CMakeLists.txt:1]: I think we should do this include only once (only in the `primary` cmake the one in src/CMakeLists.txt)

**Comment by pmoulon** [comment]
Solved by integrating #2148

**Pull request closed without merging**